tests/ConfigTest.php

```diff
@@ -193,7 +193,7 @@ public function testRegisterCustomFixersWithInvalidArgument()
         $this->expectExceptionMessageMatches('/^Argument must be an array or a Traversable, got "\w+"\.$/');
 
         $config = new Config();
-        $config->registerCustomFixers('foo');
+        $config->registerCustomFixers('foo'); // @phpstan-ignore-line to avoid `expects iterable<PhpCsFixer\Fixer\FixerInterface>, string given`
```

Comment:
I don't see the point of this change, it's moving issue handling from one place to another, while on 3.0 we already have the fix in ConfigInterface directly:
`public function registerCustomFixers(iterable $fixers): self;`

Reply:
The idea is not having such exceptions in `phpstan.neon` - this is not error to ignore, but expected wrong usage.

Reply:
my point was that whatever you produce here, it is out on 3.0 line, because we enforced proper use on typehint level

Reply:
That is not bad, is it? What's important to me it is out of `phpstan.neon` file, so that file can be improved futher.

Reply:
i'm saying that I don't see the value of this change, because what you aim is to move the warning behind this line out of phpstan.neon file, but that warning is already removed in phpstan.neon file on 3.0 line: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/phpstan.neon

so, doing the change on 2.x is not bringing any benefit (we have better change already on 3.0), but only causing the conflict for 2.x->3.0 merge.